Dashboard test is failing since release of aiida-core 2.2.0

The test on the CP2K dashboard [started failing](https://dashboard.cp2k.org/archive/aiida/index.html) after the release of [aiida-core 2.2.0](https://github.com/aiidateam/aiida-core/releases/tag/v2.2.0).

The first error in [the log](https://dashboard.cp2k.org/archive/aiida/commit_05f53a495c80b823b1fc428a2384f400b82581f6.txt) is the following:
```
_____________________ ERROR at teardown of test_validation _____________________

aiida_profile = Profile<uuid='9a3b153e13ee4af7a64a97deede33f15' name='8e402884-03e7-41b9-9a85-0e4bd964845e'>

    @pytest.fixture(scope="function")
    def clear_database(aiida_profile):
        """Clear the database after a test with this fixture"""
        yield
>       aiida_profile.reset_db()
E       AttributeError: 'Profile' object has no attribute 'reset_db'

conftest.py:18: AttributeError
```

--- a/conftest.py
+++ b/conftest.py
@@ -11,11 +11,9 @@ def cp2k_code(aiida_local_code_factory):  # pylint: disable=unused-argument
     return aiida_local_code_factory("cp2k", "cp2k")
 
 
-@pytest.fixture(scope="function")
-def clear_database(aiida_profile):
-    """Clear the database after a test with this fixture"""
-    yield
-    aiida_profile.reset_db()
+@pytest.fixture(scope="function", autouse=True)
+def clear_database(aiida_profile_clean):
+    """Automatically clear database in between tests."""
 
 
 # from https://stackoverflow.com/a/25188424

--- a/docs/source/conf.py
+++ b/docs/source/conf.py
@@ -360,4 +360,5 @@ nitpick_ignore = [
     ("py:class", "utils.AttributesFrozendict"),
     ("py:class", "State"),
     ("py:class", "Stepper"),
+    ("py:class", "CalcJobNode"),
 ]

